.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,31 +8,45 @@ assignees: ''
 ---
 
 **Describe the bug**
+
 A clear and concise description of what the bug is.
 
 **To Reproduce**
+
 Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
 **Expected behavior**
+
 A clear and concise description of what you expected to happen.
 
 **Screenshots**
+
 If applicable, add screenshots to help explain your problem.
 
 **Desktop (please complete the following information):**
+
  - OS: [e.g. iOS]
  - Browser [e.g. chrome, safari]
  - Version [e.g. 22]
 
 **Smartphone (please complete the following information):**
+
  - Device: [e.g. iPhone6]
  - OS: [e.g. iOS8.1]
  - Browser [e.g. stock browser, safari]
  - Version [e.g. 22]
 
 **Additional context**
+
 Add any other context about the problem here.
+
+### Checklist:
+
+- [ ] I have double checked that the issue does not currently exist. 
+- [ ] I have filled out the above template.
```

Comment:
I suggest we remove "I have filled out the template" from the checklist as going through the checklist means they are filling it out